.github/workflows/integration-tests.yml

```diff
@@ -80,6 +80,7 @@ jobs:
         working-directory: integration_tests/backend
         env:
           SERVER_ADDRESS: localhost:8080
+          INTEGRATION: aqueduct_demo
```

Comment:
@kenxu95 the test failed complaining about missing env var: https://github.com/aqueducthq/aqueduct/runs/7586075455?check_suite_focus=true . Should this change be merged prior to this PR?

Reply:
Hmm I thought we would be checking out the current branch so things would work ( actions/checkout@v2 is called at the top) @hsubbaraj-spiral is this true?

yeah if it always uses the main branch